data_source_examples/slack_conversation_history/using_airtable.js/.env.example

```diff
@@ -0,0 +1,4 @@
+AIRTABLE_API_KEY=keyXXXX
```

Comment:
I like the description in the readme - but you could also put comments in here (I believe # works as a comment in ENV files. (even if the comment is just to read the readme ;P)